sdk_container/src/third_party/coreos-overlay/sys-kernel/bootengine/bootengine-9999.ebuild

```diff
@@ -2,15 +2,15 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-CROS_WORKON_PROJECT="flatcar/bootengine"
+CROS_WORKON_PROJECT="simoncampion/bootengine"
 CROS_WORKON_LOCALNAME="bootengine"
 CROS_WORKON_OUTOFTREE_BUILD=1
 CROS_WORKON_REPO="https://github.com"
 
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 else
-	CROS_WORKON_COMMIT="0bade95d3b33b75b6c827d2db2f9298aff0ca05f" # flatcar-master
+	CROS_WORKON_COMMIT="32419d727df92c9212b9b7405ccb95922c8fd4d4"
```

Comment:
The PR is merged, we can use the merge commit now, and then we also need a changelog entry in `changelog/changes/` like `- Added Ignition Clevis support for encrypted root disks unlocked with a TPM2 device or a tang-server ([scripts#1560](https://github.com/flatcar/scripts/pull/1560))`

Reply:
I updated it to refer to the merge commit, and I also added a changelog entry.